### PR TITLE
predict the image class

### PR DESCRIPTION
**Pull request by ardaakcabuyuk**
WHAT I DID
------------
Add the predictor code.

HOW TO TEST
-------------
Run `python main.py`

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    "Street-light": [("Road Problem", 0.4), ("Electricity", 0.3), ("Transportation", 0.2), ("Accident", 0.1)],
+    "Traffic-light": [("Road Problem", 0.4), ("Transportation", 0.2), ("Accident", 0.4)],
+    "Traffic-sign": [("Road Problem", 0.4), ("Transportation", 0.1), ("Accident", 0.1), ("Road Maintenance", 0.4)],
+    "bus": [("Road Problem", 0.3), ("Transportation", 0.6), ("Accident", 0.1)],
+    "cyclist": [("Road Problem", 0.5), ("Transportation", 0.4), ("Accident", 0.1)],
+    "pedestrian": [("Road Problem", 0.4), ("Transportation", 0.1), ("Missing", 0.3), ("Accident", 0.2)],
+    "vehicle": [("Road Problem", 0.5), ("Transportation", 0.3), ("Accident", 0.1), ("Road Maintenance", 0.1)]
+}
+'''
+bucket_name = "reportown-347509.appspot.com"
```
Comment (elifozerr) [inline main.py]: Can you remove private credentials from here?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        cv2.imshow('Image', img)
+        cv2.waitKey(0)
+        cv2.destroyAllWindows()
+    except Exception as e:
+        print(e)
+
+    '''
+    prediction = Prediction(predictions=confidences)
+    json_data = jsonable_encoder(prediction)
+    return JSONResponse(content=json_data)
```
Comment (elifozerr) [inline main.py]: Can you remove these lines?

**Review by elifozerr — CHANGES_REQUESTED** [review]
Can you change these as soon as possible?

File: main.py (hunk 1 above)
Comment (ardaakcabuyuk) [reply main.py]: I will, thank you.

File: main.py (hunk 2 above)
Comment (ardaakcabuyuk) [reply main.py]: OK.

**Review by elifozerr — APPROVED** [review]
Thanks.

**Pull request merged**